prov/terraform/openstack/generator.go

```diff
@@ -34,9 +35,27 @@ import (
 	"github.com/ystia/yorc/v4/tosca"
 )
 
+const (
+	floatingIPEndpointCapAttribute = "/capabilities/endpoint/attributes/floating_ip_address"
```

Comment:
Maybe not for this one as so many work is already in this PR but I'm not confortable with those consul storage specific stuff being out of the deployments package... We should consider refactor it somehow.

Reply:
But I do'nt see anything wrong about it